staubli_rx160_support/urdf/rx160_macro.xacro

```diff
@@ -18,6 +26,12 @@
           <mesh filename="package://staubli_rx160_support/meshes/rx160/collision/base_link.stl"/>
         </geometry>
       </collision>
+      <inertial>
+          <mass value="${base_mass}"/>
+          <inertia ixx="0.001" ixy="0.0" ixz="0.0"
+                   iyy="0.001" iyz="0.0"
+                   izz="0.001"/>
+        </inertial>
```

Comment:
In my experience most CAD models don't include things like wiring and other such things that have an influence on mass and inertia. I agree that these values (probably) are better than the default ones, but we can't really claim they are the 'real' values, so I feel that adding the comment is still justified.

We could mention that these are probably as good as they are going to get though.